src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/IndexPageTests.kt

```diff
@@ -112,4 +111,38 @@ class IndexPageTests : SeleniumTest()
         val component = driver.findElements(By.cssSelector("#setPinnedReportsVueApp"))
         assertThat(component.count()).isEqualTo(0)
     }
+
+    @Test
+    fun `can link to run report page with permission`()
+    {
+        setUpDb()
+        startApp("auth.provider=montagu")
+
+        addUserWithPermissions(listOf(
+                ReifiedPermission("reports.run", Scope.Global())))
+
+        loginWithMontagu()
+        driver.get(RequestHelper.webBaseUrl)
+
+        val component = driver.findElements(By.cssSelector(".ml-5 > a:nth-child(1)"))
```

Comment:
This looks a little fragile - may be better just to add an ID to the relevant element